assets/javascripts/kitten/components/date-picker/index.js

```diff
@@ -0,0 +1,273 @@
+    `}
+    border-radius: 0;
+
+    &.DayPicker-Day--outside {
+      border: 0;
+    }
+  }
+
+  .DayPicker-Day--today {
+    ${TYPOGRAPHY.fontStyles.regular}
+    ${({ styles }) => css`
+      color: ${styles.day.today.color};
+    `}
+  }
+
+  .DayPicker-Day--selected {
+    &:not(.DayPicker-Day--disabled) {
+      &:not(.DayPicker-Day--outside) {
+        ${({ styles }) => css`
+          background-color: ${styles.day.selected.backgroundColor};
+          color: ${styles.day.selected.color};
+          border: double ${styles.day.selected.backgroundColor} ${borderSize};
+        `}
+        &:hover {
+          ${({ styles }) => css`
+            background-color: ${styles.day.selected.backgroundColor};
+            color: ${styles.day.selected.color};
+            border: double ${styles.day.selected.backgroundColor} ${borderSize};
+          `}
+        }
+      }
+    }
+  }
+
+  .DayPicker-Day--disabled {
+    ${({ styles }) => css`
+      color: ${styles.day.disabled.color};
+    `}
+  }
+
+  .DayPicker-Month {
+    margin: 0;
+  }
+
+  .DayPicker-wrapper {
+    padding: 0 30px 30px 30px;
+    outline: none;
+  }
+
+  .DayPicker {
+    ${fontSize}
+    ${TYPOGRAPHY.fontStyles.light}
+    outline: none;
+
+    ${({ styles }) => css`
+      border: solid ${styles.borderColor} ${borderSize};
+    `}
+    /* Hovered selectable day*/
+    &:not(.DayPicker--interactionDisabled) {
+      .DayPicker-Day {
+        &:not(.DayPicker-Day--disabled) {
+          &:not(.DayPicker-Day--selected) {
+            &:not(.DayPicker-Day--outside) {
+              &:hover {
+                ${TYPOGRAPHY.fontStyles.light}
+                ${({ styles }) => css`
+                  background-color: ${styles.day.hover.backgroundColor};
+                  color: ${styles.day.hover.color};
+                  border: double ${styles.day.hover.backgroundColor}
+                    ${borderSize};
+                `}
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+export class DatePicker extends Component {
+  static defaultProps = {
+    locale: 'fr',
+    inputIcon: '📅',
+    datePickerProps: {
+      disabledDays: [
+        {
+          after: new Date(),
+        },
+      ],
+    },
+    styles: {
+      header: {
+        backgroundColor: COLORS.font1,
+        color: '#fff',
+        icon: {
+          color: '#fff',
+        },
+      },
+      borderColor: COLORS.line1,
+      weekdaysColor: COLORS.font1,
+      day: {
+        hover: {
+          backgroundColor: COLORS.primary1,
+          color: '#fff',
+        },
+        today: {
+          color: COLORS.primary1,
+        },
+        disabled: {
+          color: COLORS.line2,
+        },
+        selected: {
+          backgroundColor: COLORS.primary1,
+          color: '#fff',
+        },
+      },
+    },
+    textInputProps: {},
+  }
+
+  state = {
+    selectedDay: new Date(),
+    focused: null,
+    locale: this.props.locale || 'en',
```

Comment:
J'ai l'impression que `this.props` n'est pas accessible ici.

Reply:
Elle l'est :), le changement de locale est bien pris en compte sur StoryBook.